ifrs17/Import/ImportStorage.ipynb

```diff
@@ -273,15 +274,8 @@
                 "\n",
                 "\n        if(DefaultPartition != TargetPartition) {",
                 "\n            await querySource.Partition.SetAsync<PartitionByReportingNodeAndPeriod>(DefaultPartition);",
-                "\n            var defaultRawVariables = await querySource.Query<RawVariable>()",
-                "\n                                                       .Where(rv => rv.Partition == DefaultPartition)",
-                "\n                                                       .Where(rv => primaryScope.Contains(rv.DataNode)).ToArrayAsync();",
-                "\n            var defaultIfrsVariables = await querySource.Query<IfrsVariable>()",
-                "\n                                                        .Where(iv => iv.Partition == DefaultPartition)",
-                "\n                                                        .Where(iv => primaryScopeFromParsedVariables.Contains(iv.DataNode)",
-                "\n                                                                     && !EstimateTypesByImportFormat[ImportFormat].Contains(iv.EstimateType)  ",
-                "\n                                                                     || primaryScopeFromLinkedReinsurance.Contains(iv.DataNode) ",
-                "\n                                                                     || secondaryScope.Contains(iv.DataNode)).ToArrayAsync();",
+                "\n            var defaultRawVariables = await querySource.Query<RawVariable>().Where(rv => rv.Partition == DefaultPartition).Where(rv => primaryScope.Contains(rv.DataNode)).ToArrayAsync();",
+                "\n            var defaultIfrsVariables = await querySource.Query<IfrsVariable>().Where(iv => iv.Partition == DefaultPartition).Where(iv => allImportScopes.Contains(iv.DataNode)).ToArrayAsync();         ",
```

Comment:
@dcolleoni  Instead of saving all zeros, we need to store only 0 value variable that are in the intersection between the BE and the computed SENS ifrsVariable sets

Reply:
why do you write this comment here in the ImportStorage?

Reply:
this is now implemented